tests/generated_results/inplace_risk_test.xml

```diff
@@ -187,4 +187,4 @@
     <ns0:score maximum="100.000000" system="urn:xccdf:scoring:default">49.081196</ns0:score>
     <ns0:score maximum="70.000000" system="urn:xccdf:scoring:flat">40.000000</ns0:score>
   </ns0:TestResult>
-</ns0:Benchmark>
+</ns0:Benchmark>
```

Comment:
This is a whitespace noise.

Note that the UNIX convention is that all text files should have newline at the end. Whoever committed this needs to fix their editor, otherwise the next person touching the file (with proper editor) will see exactly opposite hunk.